src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js

```diff
@@ -66,7 +66,12 @@
                 endpoint.isConnected = true;
                 fillDisplayValuesForEndpoint(endpoint);
                 if (index >= 0) {
+                    var serviceControlId = $scope.endpoints[index].serviceControlId;
```

Comment:
It took me a while to work out what was going on here. Can we maybe rename the variables to `previousServiceControlId` and `previousErrorCount` or something else that represents the intention a bit clearer?

Reply:
👍 for renaming